src/schemas/json/project.json

```diff
@@ -655,6 +655,10 @@
 		"runtimes": {
 			"description": "List of runtime identifiers supported by this project (used when building standalone applications).",
 			"type": "object"
+		},
+		"userSecretsId": {
+			"description": "Specify a unique id to manage the development time user secrets of the application.",
```

Comment:
Good points. Saying "of the application" was for indicating that each project has its own unique identifier.